src/main/java/xyz/groundx/caver_ext_kas/kas/wallet/Wallet.java

```diff
@@ -3344,7 +3401,216 @@ public KeyStatus deleteKey(String keyId) throws ApiException {
+     * }</pre>
+     *
+     * @param callback The callback to handle response
+     * @return Call
+     * @throws ApiException
+     */
+    public Call getFDTransactionListAsync(ApiCallback<FDTransactionWithCurrencyResultList> callback) throws ApiException {
+        return getFDTransactionListAsync(null, callback);
+    }
+
+    /**
+     * Retrieve the histories of fee delegation transactions.<br>
+     * You can find out the KRW and USD price of the fees at the time of sending the transaction.<br>
+     * If you passed from passed as a parameter, retrieves transaction only from a certain address.<br>
+     * GET /v2/history/fd/tx
+     *
+     * <pre>{@code
+     * String from = "0x{fromAddress}";
+     * FDTransactionWithCurrencyResultList response = caver.wallet.getFDTransactionList(from);
+     * }</pre>
+     *
+     * @param from The Klaytn account address of the sender.
+     * @return FDTransactionWithCurrencyResultList
+     * @throws ApiException
+     */
+    public FDTransactionWithCurrencyResultList getFDTransactionList(String from) throws ApiException {
+        return this.transactionHistoryApi.getV2HistoryFdTx(chainId, from);
+    }
+
+    /**
+     * Retrieve the histories of fee delegated transactions asynchronously.<br>
+     * You can find out the KRW and USD price of the fees at the time of sending the transaction.<br>
+     * If you passed from passed as a parameter, retrieves transaction only from a certain address.<br>
+     * GET /v2/history/fd/tx
+     *
+     * <pre>{@code
+     * ApiCallback<FDTransactionWithCurrencyResultList> callback = new ApiCallback<FDTransactionWithCurrencyResultList> callback() {
+     *   ....implement callback method.
+     * };
+     *
+     * String from = "0x{fromAddress}";
+     * caver.wallet.getFDTransactionListAsync(from, callback);
+     * }</pre>
+     *
+     * @param from The Klaytn account address of the sender.
+     * @param callback The callback to handle response.
+     * @return Call
+     * @throws ApiException
+     */
+    public Call getFDTransactionListAsync(String from, ApiCallback<FDTransactionWithCurrencyResultList> callback) throws ApiException {
+        return this.transactionHistoryApi.getV2HistoryFdTxAsync(chainId, from, callback);
+    }
+
+    /**
+     * Retrieves a fee delegated transaction by transaction hash.<br>
+     * You can find out the KRW and USD price of the fees at the time of sending the transaction.<br>
+     * GET /v2/history/fd/tx/{transaction-hash}
+     *
+     * <pre>{@code
+     * String txHash = "0x{transactionHash}";
+     * FDTransactionWithCurrencyResult response = caver.wallet.getFDTransaction(txHash);
+     * }</pre>
+     *
+     * @param txHash The transaction hash to get a fee delegated transaction.
+     * @return FDTransactionWithCurrencyResult
+     * @throws ApiException
+     */
+    public FDTransactionWithCurrencyResult getFDTransaction(String txHash) throws ApiException {
```

Comment:
As above, it would be good if a method for users to know that it is an API that only works with cypress would be added.

Reply:
Please check chain id for this too.